plugins/git/git.plugin.zsh

```diff
@@ -215,3 +215,8 @@ alias glum='git pull upstream master'
 
 alias gwch='git whatchanged -p --abbrev-commit --pretty=medium'
 alias gwip='git add -A; git rm $(git ls-files --deleted) 2> /dev/null; git commit -m "--wip--"'
+
+# Remove from repo all files deleted
+alias gitstrm="git st | egrep 'deleted' | sed -e 's/#//' -e 's/deleted:/git rm/' | sh"
```

Comment:
What does this do? `git st` I guess stands for `git status`?

Ok, this runs `git rm` on all files deleted from working space. I'm sure there's a better way of doing that: [`git ls-files --deleted -z | xargs -0 git rm`](http://stackoverflow.com/a/5147119/5798232).